Doubts/Watchers.html

```diff
@@ -0,0 +1,33 @@
+<html>
+
+<head>
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+
+</head>
+
+<body>
+    <div id="demo1">Watch: {{ fullName }}</div>
+
+
+    <script>
+        var vm = new Vue({
+            el: '#demo1',
+            data: {
+                firstName: 'Abhay',
+                lastName: 'Mehta',
+                fullName: 'Abhay Mehta'
```

Comment:
you have already done what watchers intend to do,
you need to put it as empty or maybe something other than the result itself